tests/tensorflow/r2.15/common.libsonnet

```diff
@@ -0,0 +1,359 @@
+            local tpuCreateSettings = {
+              acceleratorName: std.escapeStringBash(config.accelerator.name),
+              softwareVersion: std.escapeStringBash(config.tpuSettings.softwareVersion),
+              startupScript: std.escapeStringBash(config.tpuSettings.tpuVmStartupScript),
+              sleepTime: config.tpuSettings.tpuVmCreateSleepSeconds,
+              testName: std.strReplace(config.testName, '.', '-'),
+            },
+            command: utils.scriptCommand(|||
+              project=$(curl -sS "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
+              zone=$(curl -sS "http://metadata.google.internal/computeMetadata/v1/instance/zone" -H "Metadata-Flavor: Google" | awk -F'/' '{print $4}')
+              tpu_name=tpu-${POD_UID}
+              ssh-keygen -t rsa -f /scripts/id_rsa -q -N ""
+
+              echo "
+              gcloud alpha compute tpus tpu-vm delete -q --async ${tpu_name} --zone=${zone}
+              sleep 60
+              " > /scripts/cleanup.sh
+
+              echo "xl-ml-test:$(cat /scripts/id_rsa.pub)" > ssh-keys.txt
+              echo %(startupScript)s > startup-script.txt
+
+              # Retry every 30 seconds for up to 10 minutes
+              start_time="$(date -u +%%s)"
+              for i in {1..20}; do
+                set +e
+                gcloud alpha compute tpus tpu-vm create ${tpu_name} \
+                  --accelerator-type=%(acceleratorName)s \
+                  --version=%(softwareVersion)s  \
+                  --metadata-from-file='ssh-keys=ssh-keys.txt,startup-script=startup-script.txt' \
+                  --labels='test-name=%(testName)s' \
+                  --zone=${zone}
+
+                exit_code=$?
+                set -e
+
+                current_time="$(date -u +%%s)"
+                elapsed_seconds=$(($current_time-$start_time))
+                # Break if command passed or 10-minute limit reached
+                test $exit_code = 0 && break
+                test $elapsed_seconds -gt 600 && break
+                sleep 30
+              done
+
+              if [ $exit_code -ne 0 ]; then
+                exit $exit_code
+              fi
+
+
+              echo ${zone} > /scripts/zone
+              echo ${tpu_name} > /scripts/tpu_name
+              gcloud compute tpus describe ${tpu_name} --project=${project} --zone=${zone} --format="value(networkEndpoints[0].ipAddress)" > /scripts/tpu_ip
+              gcloud compute tpus describe ${tpu_name} --project=${project} --zone=${zone} --flatten="networkEndpoints[]" --format="csv[no-heading](networkEndpoints.ipAddress)" > /scripts/all_tpu_ips
+              sleep %(sleepTime)d
+
+              softwareVersion=%(softwareVersion)s
+              gcloud alpha compute tpus tpu-vm ssh ${tpu_name}  --zone=${zone} --project=${project}  --internal-ip --ssh-key-file=/scripts/id_rsa --worker=0 --command "pip install tensorflow-text-nightly"
```

Comment:
pip install tensorflow-text==2.15.0rc0. But we're still waiting for it to release. So you can comment it for now.